src/GovukNotify/Models/Notification.cs

```diff
@@ -59,18 +67,18 @@ public override bool Equals(object notification)
                 line4 == note.line4 &&
                 line5 == note.line5 &&
                 line6 == note.line6 &&
+                line7 == note.line7 &&
                 phoneNumber == note.phoneNumber &&
-                postcode == note.postcode &&
```

Comment:
Why is this and the template below being removed?

Reply:
thanks I had removed it to test something and pushed the change without realising.. a bit worrying that the tests passed

https://github.com/alphagov/notifications-net-client/pull/188/commits/b0865c729056a719be05cd7a1da3df9edabd2054